docs/rules/no-extraneous-dependencies.md

```diff
@@ -15,10 +15,20 @@ This rule supports the following options:
 
 `peerDependencies`: If set to `false`, then the rule will show an error when `peerDependencies` are imported. Defaults to `false`.
 
+`ignore`:  Some modules can be ignored by the ignore parameter.It supports regular expressions
```

Comment:
I’m pretty opposed to options that are regexes; globs would be sufficient.

Separately:
```suggestion
`ignore`:  Some modules can be ignored by the `ignore` parameter. It supports regular expressions.
```